### DISCUSSION TITLE
Sync other leds with hyperhdr

### DISCUSSION THREAD

**Discussion by M20200085622** [awawa-dev/HyperHDR#820] [Q&A]
im starting with hyperhdr now. and wanted to know if is possible sync other leds strips with hyperhdr
my setup is 1 esp8266, 1 splitter, 1 video grabber, 1 led strip WS2812BECO and 1 pi 4.

**Comment by awawa-dev** [comment]
Hi
WS2812B is supported by WLED/HyperSerialEsp8266 on esp8266 with HyperHDR. Recommended LED strip is sk6812 RGBW and Pico(rp2040 preferable with built-in level shifter) with HyperSerialPico.
